index.html

```diff
@@ -249,79 +249,74 @@ <h1>Introduction</h1>
-Resource Locator).
+numbers, barcodes, RFIDs). URIs (Uniform Resource Identifiers) are used for
+resources on the Web and each web page you view in a browser has a globally
+unique URL (Uniform Resource Locator).
     </p>
     <p>
 The vast majority of these globally unique identifiers are not under our
 control. They are issued by external authorities that decide who or what they
 identify and when they can be revoked. They are useful only in certain contexts
-and recognized only by certain bodies (not of our choosing). They may disappear
-or cease to be valid with the failure of an organization. They may unnecessarily
-reveal personal information. And in many cases they can be fraudulently
-replicated and asserted by a malicious third-party ("identity theft").
+and recognized only by certain bodies not of our choosing. They might
+disappear or cease to be valid with the failure of an organization. They might
+unnecessarily reveal personal information. In many cases, they can be
+fraudulently replicated and asserted by a malicious third-party, which is
+more commonly known as "identity theft".
     </p>
     <p>
 The Decentralized Identifiers (DIDs) defined in this specification are a new
-type of globally unique identifier designed to enable individuals and
-organizations to generate our own identifiers using systems we trust, and to
-prove control of those identifiers (authenticate) using cryptographic proofs
-(for example, digital signatures).
+type of globally unique identifier. They are designed to enable individuals and
+organizations to generate our own identifiers using systems we trust. These
+new identifiers enable us to prove control of over them by authenticating
+using cryptographic proofs such as digital signatures.
     </p>
     <p>
-Because we control the generation and assertion of these identifiers, each of us
-can have as many DIDs as we need to respect our desired separation of
-identities, personas, and contexts (in the everyday sense of these words). We
-can scope the use of these identifiers to the most appropriate contexts. We can
-interact with other people, institutions or systems that require us to identify
-ourselves (or things we control) while maintaining control over how much
-personal or private data should be revealed, and without depending on a central
-authority to guarantee the continued existence of the identifier.
+Since we control the generation and assertion of Decentralized Identifiers, each
+of us can have as many DIDs as we need to respect our desired separation of
+identities, personas, and interactions. We can scope the use of these
+identifiers to the most appropriate contexts. We can interact with other people,
+institutions or systems that require us to identify ourselves, or things we
+control, while maintaining control over how much personal or private data should
+be revealed, and without depending on a central authority to guarantee the
+continued existence of the identifier.
```

Comment:
```suggestion
Since the generation and assertion of Decentralized Identifiers is
user-controlled, each user can have as many DIDs as they need to respect a
desired separation of identities, personas, and interactions. The use of these
identifiers can be scoped appropriately to different contexts. They support
interactions with other people, institutions, or systems that require users to
identify themselves, or things they control, while providing control over how
much personal or private data should be revealed, all without depending on a
central authority to guarantee the continued existence of the identifier.
```

Reply:
In line 275, here, I'd change `as many DIDs as they need to respect a` to `as many DIDs as necessary to maintain their`...

Reply:
Changed user->entity. Applied @TallTed's suggestion.